tests/testcases.conf

```diff
@@ -185,6 +185,18 @@ section "Tests with id"
 id "SWR_001" cukinia_cmd true
 id "SWR_002" as "Running cukinia command with as and id" cukinia_cmd true
 
+section "on failure retry"
+on failure retry 3 after 1 as "Should fail after 3 retries"  cukinia_cmd /bin/false
```

Comment:
Suggest adding a `tests/flaky`:
```
#!/bin/sh

# Get the current second number
second=$(date +%S)

# Check if the second number is even
if [ $((second % 2)) -eq 0 ]; then
    exit 0  # Success
else
    exit 1  # Failure
fi
```

with it, I get:
```
$ ./cukinia tmp
Retry "Will pass on first try, or on retry" in 1s second(s)
[PASS]  Will pass on first try, or on retry
```
or
```
$ ./cukinia tmp
[PASS]  Will pass on first try, or on retry
```

where `tmp` contains `on failure retry 1 after 1s as "Will pass on first try, or on retry" cukinia_cmd ./tests/flaky`.